File: RoyalDragon.Admin/Client/src/app/api/models/init-home-page-response.ts
/* tslint:disable */
/* eslint-disable */
import { ConfigurationParameters } from './configuration-parameters';
import { ProductCustom } from './product-custom';
import { VListCategoryHomePage } from './v-list-category-home-page';
export interface InitHomePageResponse {
  banner?: null | Array<ProductCustom>;
  feather?: null | Array<ProductCustom>;
  listCategory?: null | Array<VListCategoryHomePage>;
  listConfigurationParameters?: null | Array<ConfigurationParameters>;
  popular?: null | Array<ProductCustom>;
  specialOffer?: null | Array<ProductCustom>;
}

File: RoyalDragon.Admin/Models/HomePageModels/InitHomePageResponse.cs
using RoyalDragon.Admin.Models.EcommerceModels;
using RoyalDragon.Admin.Models.Entities;

namespace RoyalDragon.Admin.Models.HomePageModels
{
    public class InitHomePageResponse
    {
        public IEnumerable<ProductCustom> Banner { get; set; }
        public IEnumerable<ProductCustom> Feather { get; set; }
        public IEnumerable<ProductCustom> SpecialOffer { get; set; }
        public IEnumerable<ProductCustom> Popular { get; set; }
        public IEnumerable<VListCategoryHomePage> ListCategory { get; set; }
        public IEnumerable<ConfigurationParameters> ListConfigurationParameters { get; set; }
    }
}
